docs/pages/development/create-development-builds.mdx

```diff
@@ -0,0 +1,208 @@
+
+<Tab>
+
+> If you have created a development build for an Android device, you do not need to create it separately the emulator. You can skip this step since the same **.apk** will work in both scenarios.
+
+To create and install the development build on an Android Emulator, you will need a **.apk**. To create it, run the following command:
+
+<Terminal cmd={['$ eas build --profile development --platform android']} />
+
+After the build is complete, go to the build page in the [Expo dashboard](https://expo.dev/accounts/[account]/projects/[project]/builds) or the open link to it provided when `eas build` has finished running the process. Then, open the **Options** dropdown menu, and click the **Download build** button to download the **.apk**.
+
+<ImageSpotlight
+  alt="Open the options dropdown menu to install the apk file."
+  src="/static/images/dev-client/android-build.jpg"
+  style={{ maxWidth: 700 }}
+/>
+
+You can now install it on the Android Emulator by dragging **.apk** file into the emulator. When it's complete installing, navigate to the app launcher, find the app icon and open it.
+
+</Tab>
+
+<Tab>
+
+To create and install a development build on an iOS simulator, we recommend you to create a separate [build profile](/build/eas-json/#build-profiles) for the simulator and then set the `ios.simulator` option to `true` in the **eas.json**.
+
+For example, the `"development-simulator"` profile below is only for creating a development build for iOS simulator:
+
+```json eas.json
+{
+  "builds": {
+    "development-simulator": {
+      "developmentClient": true,
+      "distribution": "internal",
+      "ios": {
+        "simulator": true
+      }
+    }
+  }
+}
+```
+
+Then, run the following command to create the development build on an iOS simulator:
+
+<Terminal cmd={['$ eas build --profile development-simulator --platform ios']} />
+
+> Make sure to use that profile name as the argument for the `--profile` flag in the above command.
+
+After the build is complete, go to the build page in the [Expo dashboard](https://expo.dev/accounts/[account]/projects/[project]/builds) and click the **Download** button.
+
+<ImageSpotlight
+  alt="Click the download button to download the build for iOS simulator."
+  src="/static/images/dev-client/ios-build.jpg"
+  style={{ maxWidth: 700 }}
+/>
+
+You can now install the it on an iOS simulator:
+
+- Open the **.tar.gz** file to extract it. You will now have a **.app** file, for example, **myapp.app**.
+- Drag the **.app** file into the iOS simulator. When you see it appear on the simulator home screen, tap on it to run it.
+
+</Tab>
+
+</Tabs>
+
+## Build locally with Android Studio and Xcode
+
+If you are comfortable setting up or have access to Android, Xcode, and related dependencies, you can build and distribute your project the same as any other Android or iOS app. You will still need to [install `expo-dev-client`](#install-expo-dev-client) in your project.
```

Comment:
At least a few times, we've had folks who have gone to this step when they would have been just fine/ really should have used EAS, so would like to have a stronger reason as to why this road should be taken (so folks who don't need it are strongly disincentivized).

Maybe it's something like, "If you need to debug native code, you can build your project the same..." with an Info blurb reminding folks that they usually don't need to do this.

Reply:
I did add the reasons at the end of the section:

```
We do not recommend modifying these directories directly, especially when creating a development build with EAS Build. However, you can use this method when you:

- Want to debug the native code directly
- You are managing your own native projects
```

Do you think leading with them will be better rather than having them at the end of this section as we don't want folks to use this route?

Reply:
I think if we have this here, we really need to lead with "you shouldn't need this, but...". The current lead-in almost dares folks to try it ("Xcode? Oh yeah, I'm so good at that, let's do this!")

Which leads me to... do we really need this section at all? We already have a Bare Workflow guide page for dev builds; the only ground not covered here is that the bare workflow guide assumes you're adding dev builds to a vanilla RN app instead of an Expo app where you ran `prebuild` first.

`npx expo run` is [already covered in the Expo CLI section](https://www.loom.com/share/ca4c161aaaf74e9fb12f78e21a11170a) and has good verbiage for describing why most folks don't need this. I'm not aware of "but will development builds really work with `npx expo run`?" being a burning question, and nothing implies it will not work with it, so IMO you could delete this section entirely.